scripts/ci/docker-compose/integration-kerberos.yml

```diff
@@ -17,16 +17,55 @@
 ---
 version: "2.2"
 services:
-  kerberos:
-    image: godatadriven/krb5-kdc-server
-    hostname: kerberos
+  kdc-server-example-com:
+    build:
+      context: ..
+      dockerfile: ./docker-files/kdc-server/Dockerfile
+    image: krb5-kdc-server-example-com
+    hostname: krb5-kdc-server-example-com
     domainname: example.com
+    networks:
+      example.com:
+        ipv4_address: 10.5.0.2
+
     volumes:
+      - kerberos-keytabs:/root/kerberos-keytabs
       - /dev/urandom:/dev/random   # Required to get non-blocking entropy source
+
+    environment:
+      - KRB5_TRACE=/dev/stderr
+      - POST_BOOTSTRAP_COMMAND=
+        /opt/kerberos-utils/create_admin.sh alice alice;
+        /opt/kerberos-utils/create_client.sh bob bob /root/kerberos-keytabs/airflow.keytab;
```

Comment:
This is my favorite part, it's very easy now to set up more services and clients as we keep adding more integrations.